cmd/gcs/main.go

```diff
@@ -196,7 +196,7 @@ func main() {
 		"If true do not run chronyd time synchronization service inside the UVM")
 	scrubLogs := flag.Bool("scrub-logs", false, "If true, scrub potentially sensitive information from logging")
 	initialPolicyStance := flag.String("initial-policy-stance",
-		"deny",
+		"allow",
```

Comment:
What is the impact of switching to "allow"?

Reply:
If the default is `deny` in a non-confidential case logrus will log to io.Discard and we may miss a few warning and highler level kmsg logs (for example), which can be important. For confidential we'll always default to `deny`.